styles/components/_category.scss

```diff
@@ -0,0 +1,67 @@
+.category {
+    display: flex;
+    justify-content: space-evenly;
+    position: relative;
+
+    &__name {
+        display: flex;
+        justify-content: space-evenly;
+        margin-top: 180px;
+        flex-wrap: nowrap;
+    }
+
+    &__text-primary {
+        align-items: center;
+        text-decoration: none;
+    }
+
+    &__text-secondary {
+        text-decoration: none;
+    }
+
+    &__mall {
+        padding: 0px 30px;
+        padding-bottom: 140px;
+        background-color: #fefeff;
+        border-bottom-left-radius: 25px;
+        border-bottom-right-radius: 25px;
+        box-shadow: 20px 20px 20px 20px var(--color-text-fifth);
+    }
+
+    &__text-primary {
+        align-items: center;
+        color: var(--color-text-secondary);
+        font-weight: bold 3px;
+        font-size: 20px;
+        position: relative;
+        top: 35px;
+        left: 20px;
```

Comment:
check in google, how to use position and top, left, right, bottom? I guided u from yesterday